### DISCUSSION TITLE
New zoom out feature when tap on screen

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#59] [Q&A] [answered]
When tapping on the display in navigation mode, the buttons appear. Could then automatically a zoom out by let's say 20% be done at the same time? By doing this you could see the next 10 to 20km of the route. After the timer which is already available normal zoom again. 
The complete route overview which is available on compass has the disadvantage that you see in principle nothing on a long route. Better would be to just see a greater section from actual gps position. To my opinion this would be much more useful and be a good function for an action button.

**Comment by devemux86** [comment, marked as answer]
Thanks for the suggestion.

You can zoom out by tapping on the screen with 2 fingers.
